LearningMachine/app/src/main/java/com/learningmachine/android/app/ui/cert/CertificateFragment.java

```diff
@@ -154,34 +158,61 @@ private String displayHTML(BlockCert certificate) {
-        String customCss = "body {padding: 20px; font-size: 12px; line-height: 1.5;} body > section { padding: 0;} body section { max-width: 100%; } body img { max-width: 100%; height: auto; width: inherit; }";
-        String wrappedHtml = String.format("<!doctype html><html class=\"no-js\" lang=\"\"><head><meta charset=\"utf-8\"><meta http-equiv=\"x-ua-compatible\" content=\"ie=edge\"><title></title><meta content=\"width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0\" name=\"viewport\" /><meta name=”viewport” content=”width=device-width” /><style type=\"text/css\">%s</style><style type=\"text/css\">%s</style></head><body>%s</body></html>", normalizeCss, customCss, displayHTML);
+        String customCss = "body { font-size: 12px; line-height: 1.5; margin:20px; display: inline-block; } body > section { padding: 0; } body section { max-width: 100%; word-break: break-word; } body img { max-width: 100%; height: auto; width: inherit; }";
+        String autolinkerScript = getAutolinkerScript();
+        return String.format("<!doctype html><html class=\"no-js\" lang=\"\"><head><meta charset=\"utf-8\"><meta http-equiv=\"x-ua-compatible\" content=\"ie=edge\"><title></title><meta content=\"width=device-width, initial-scale=1.0%s\" name=\"viewport\" /><meta name=”viewport” content=”width=device-width” /><style type=\"text/css\">%s</style><style type=\"text/css\">%s</style></head><body><div id=\"_displayHTML\">%s</div></script><script type=\"text/javascript\">%s</script></body></html>", scalingMetaContent, normalizeCss, customCss, displayHTML, autolinkerScript);
+    }
 
-        return wrappedHtml;
+    private String getAutolinkerScript() {
+        String script = "";
+        try {
+            FileUtils.copyAssetFile(getContext(), "www/Autolinker.min.js", "Autolinker.js");
```

Comment:
probably document how to update Autolinker in a readme just like we have for cert-verifier-js